include/geos/noding/NodingIntersectionFinder.h

```diff
@@ -60,8 +60,7 @@ class NodingIntersectionFinder: public SegmentIntersector {
         interiorIntersection(geom::Coordinate::getNull()),
         intersectionCount(0),
         isCheckEndSegmentsOnly(false),
-        findAllIntersections(false),
-        keepIntersections(true)
```

Comment:
Probably best resolved at https://github.com/libgeos/geos/blob/master/src/noding/NodingIntersectionFinder.cpp#L149, but I guess no harm removing it in the meantime.